deployment/packer/install-nvidia-drivers.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -ex
+
+# Update nvidia drivers
+NVIDIA_DRIVERS_VERSION="375.51"
+
+wget "http://us.download.nvidia.com/XFree86/Linux-x86_64/${NVIDIA_DRIVERS_VERSION}/NVIDIA-Linux-x86_64-${NVIDIA_DRIVERS_VERSION}.run"
+sudo /bin/bash "./NVIDIA-Linux-x86_64-${NVIDIA_DRIVERS_VERSION}.run" -asq
+sudo reboot
```

Comment:
Nope, the shell provisioner's [`expect_disconnect`](https://www.packer.io/docs/provisioners/shell.html#expect_disconnect) parameter allows Packer to handle reboots. Packer will reconnect to the instance after waiting for a bit, and run the next provisioner.